scripts/index.js

```diff
@@ -0,0 +1,73 @@
+// Variables that are representing Elements
+let btnAdd = document.getElementById('add');
+let btnSubtract = document.getElementById('subtract');
+let tempInput = document.getElementById('temp_id');
+let landscape = document.getElementById('landscape');
+let skiesInput = document.getElementById('sky-select');
+let skiesLandscape = document.getElementById('sky-landscape');
```

Comment:
Not a bad idea having these at the top, but it's possible the HTML might get loaded by the browser AFTER the JS.  So in that case these variables would be set to `undefined`.  

Also why not make them `const`?